.github/workflows/deploy_docs.yml

```diff
@@ -43,6 +43,9 @@ jobs:
       - name: Deploy docs
         uses: JamesIves/github-pages-deploy-action@v4
         with:
-          branch: deploy-doc-site
+          token: ${{ secrets.GH_PAT }}
```

Comment:
@ppebay @lifflander This will require accessing LBAF's secret settings (unless we already have a PAT token set up--I copied `${{ secrets.GH_PAT }}` from the magistrate repo in case we already have one here and the syntax is the same).